autopush/router/simple.py

```diff
@@ -137,9 +145,27 @@ def route_notification(self, notification, uaid_data):
             returnValue(RouterResponse(response_body="Delivered"))
         else:
             self.metrics.increment("router.broadcast.miss")
+            if self.udp is not None:
+                yield deferToThread(self._send_udp_wake,
+                                    self.udp)
             returnValue(RouterResponse(202, "Notification Stored"))
 
-    #############################################################
+    def _send_udp_wake(self, udp_info):
```

Comment:
Heh, the first approach I took was to do that, however I realized that i had to break isolation when I did the timeout stuff. Plus the routers tend to think more in "This is a total hand-off including data" where the UDP stuff is more a side-band wake up routine.